presto-main/src/main/java/com/facebook/presto/sql/planner/TranslationMap.java

```diff
@@ -120,29 +120,13 @@ public Expression rewriteExpression(Expression node, Void context, ExpressionTre
                 if (expressionToSymbols.containsKey(node)) {
                     return expressionToSymbols.get(node).toSymbolReference();
                 }
-                else if (expressionToExpressions.containsKey(node)) {
-                    Expression mapping = getMapping(node);
```

Comment:
> Remove unnecessary method

why the method was unnecessary? the method did recursive conversion, new code doesn't do that.

Reply:
It's unnecessary because all it does is translate names -> symbols (which is done above, before the rewrite) and a lookup for each sub-expression, which is already being done by this rewriter as it walks the tree recursively. I should've added an explanation to the commit message.